client/src/app/modules/alta-main-routing/alta-main-routing.module.ts

```diff
@@ -1,18 +1,31 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { AuthTabsComponent } from 'src/app/components/auth-tabs/auth-tabs.component';
+import { DashboardComponent } from 'src/app/components/dashboard/dashboard.component';
 import { HomeComponent } from 'src/app/components/home/home.component';
+import { ManageMembersComponent } from 'src/app/components/manage-members/manage-members.component';
 import { AuthGuard } from '../../guards/auth.guard';
 
 const routes: Routes = [
+  { path: 'login', component: AuthTabsComponent },
```

Comment:
As mentioned above, I think it would be better to recreate the file responsible for routing for authentication stuff. So when you do that please add this route in the other file.